examples/shader/compute_shader_game_of_life.rs

```diff
@@ -106,7 +109,7 @@ fn prepare_bind_group(
     game_of_life_image: Res<GameOfLifeImage>,
     render_device: Res<RenderDevice>,
 ) {
-    let view = gpu_images.get(&game_of_life_image.0).unwrap();
+    let view = gpu_images.get(&game_of_life_image.texture).unwrap();
     let bind_group = render_device.create_bind_group(&BindGroupDescriptor {
```

Comment:
You could probably also use the `AsBindGroup::bind_group()` here

Reply:
That would require me getting the `FallbackImage` from somewhere, is that readily available?

Reply:
you can add `fallback_image: Res<FallbackImage>,` as a system parameter and then use it